src/systemctl/systemctl-edit.c

```diff
@@ -247,13 +247,12 @@ static int find_paths_to_edit(
                         return r; /* Already logged by unit_find_paths() */
 
                 if (!path) {
-                        if (!arg_force) {
-                                log_info("Run 'systemctl edit%s --force --full %s' to create a new unit.",
-                                         arg_runtime_scope == RUNTIME_SCOPE_GLOBAL ? " --global" :
-                                         arg_runtime_scope == RUNTIME_SCOPE_USER ? " --user" : "",
-                                         *name);
-                                return -ENOENT;
-                        }
+                        if (!arg_force)
+                                return log_info_errno(SYNTHETIC_ERRNO(ENOENT),
+                                                      "Run 'systemctl edit%s --force --full %s' to create a new unit.",
+                                                      arg_runtime_scope == RUNTIME_SCOPE_GLOBAL ? " --global" :
```

Comment:
While at it, could you please switch this to `runtime_scope_cmdline_option_to_string` too?

Reply:
Wouldn't this then print unnecessary `--system` when using the default (system) scope? I.e.

```
$ build/systemctl edit sdfasdfa
No files found for sdfasdfa.service.
Run 'systemctl edit --system --force --full sdfasdfa.service' to create a new unit.
``` 

I mean, I don't mind it, it's just slightly inconsistent with other places where we try to hide this, like

https://github.com/systemd/systemd/blob/96f69497e316f8ffb8fc8ca0bab9fb5b3b854aa0/src/systemctl/systemctl-edit.c#L93

Reply:
Hmm, I like the previous version then...

Reply:
I pushed the previous version again, i.e. the version that only adds `SYNTHETHIC_ERRNO()`. We can follow-up this in another PR once we agree on the format.

Reply:
Yeah, let's do that later.